src/DateRange/DateRange.tsx

```diff
@@ -68,16 +62,27 @@ const DateRange = ({ onChange, caption, value, className }: IProps) => {
 	const [shouldShowFilter, setShouldShowFilter] = React.useState<boolean>(
 		false
 	);
-	const hoursFrom = dateFrom.getHours();
-	const minutesFrom = dateFrom.getMinutes();
-	const hoursTo = dateTo.getHours();
-	const minutesTo = dateTo.getMinutes();
 
 	const handleSelectedNoFilter = () => {
+		setDateFrom(undefined)
```

Comment:
Что-то притиер похоже не работает
```suggestion
		setDateFrom(undefined);
```